### ISSUE TITLE
I cannot open my Keepass2Android file.

### ISSUE THREAD

**Issue by wrenchman5** [PhilippC/keepass2android#2541]
### Checks

- [X] I have read the FAQ section, searched the open issues, and still think this is a new bug.

### Describe the bug you encountered:

I've been running Keepass2Android on my Motorola Droid with Android version 11 for years with no problems. Suddenly I am not able to open my database.  On the log in screen I get the message, "Could not access source file. No content provider: content://com.google.android.apps.nbu.files.provider/2/4800. Loaded for from internal cache. You can still make changes in the database and synchronize them later." Then after entering my password, I get a second message, "An error occurred: The file signature is invalid. Either the file isn't a keepass data file at all or it is corrupted.". 

Any suggestions?

### Describe what you expected to happen:

Please provide instructions on how to regain access to my database.

### What version of Keepass2Android are you using?

Updated to 1.10-pre to try and fix the issue. Previously 1.09e

### Which version of Android are you on?

Android 11

**Comment by PhilippC** [comment]
please select "Change database", then enable the switch "Show local backups" and select the backup corresponding to your database file. Can you open this?

**Comment by wrenchman5** [comment]
Yes. That solved the problem. Thank you so much!  I was really concerned
that I had lost it.

Should I also be backing it up to another location or has this method of
back up normally been sufficient?

Thanks again, John

**Comment by gtedavid** [comment]
hi
I'm having the same issue on Android 13 where it is no longer possible to open kdbx files.

the app sees the folder as empty and I can only navigate on the phone's storage and not the sd card.

This wasn't the case earlier in the year...

**Comment by Harry2024B** [comment]
And I have the same issue with android 14

**Comment by AnAnalogGuy** [comment]
obsolete:

> Same issue here with Android 14.
> My keepass file is quiet large and is synced.

**Comment by PhilippC** [comment]
@wrenchman5 please read https://github.com/PhilippC/keepass2android/wiki/Documentation#doesnt-keepass2android-create-automatic-backups and https://github.com/PhilippC/keepass2android/wiki/Documentation#how-do-i-backup-the-database.

@gtedavid @Harry2024B  @AnAnalogGuy: When you say you have the same issue, do you mean the "An error occurred: The file signature is invalid. Either the file isn't a keepass data file at all or it is corrupted." message? Or the "empty folder" issue described by @gtedavid ?

@gtedavid I mean "Change database" on the screen where you would usually enter your password. If you tap this, you should see the list of recently opened files. There you should then be able to enable "Show local backups" (if you have any).

@gtedavid Can you post a screenshot where the SD card is missing? Please also try to find options like "Show SD card" (this exists on some Android file browsers!)

**Comment by gtedavid** [comment]
> @wrenchman5 please read https://github.com/PhilippC/keepass2android/wiki/Documentation#doesnt-keepass2android-create-automatic-backups and https://github.com/PhilippC/keepass2android/wiki/Documentation#how-do-i-backup-the-database.
> 
> @gtedavid @Harry2024B  @AnAnalogGuy: When you say you have the same issue, do you mean the "An error occurred: The file signature is invalid. Either the file isn't a keepass data file at all or it is corrupted." message? Or the "empty folder" issue described by @gtedavid ?
> 
> @gtedavid I mean "Change database" on the screen where you would usually enter your password. If you tap this, you should see the list of recently opened files. There you should then be able to enable "Show local backups" (if you have any).
> 
> @gtedavid Can you post a screenshot where the SD card is missing? Please also try to find options like "Show SD card" (this exists on some Android file browsers!)

Hi PhilippC 
It's been a while since this has happened unfortunately.
I'll try to see if I have screens.
What's sure is that I lost a consequently important database.
But also partially my fault for not backing it up elsewhere.

From memory, the interface looked weird, buggy and different then usual.
I can't remember what I did to get it back to normal, but lost my data in the process.

**Comment by AnAnalogGuy** [comment]
> [@gtedavid](https://github.com/gtedavid) [@Harry2024B](https://github.com/Harry2024B) [@AnAnalogGuy](https://github.com/AnAnalogGuy): When you say you have the same issue, do you mean the "An error occurred: The file signature is invalid. Either the file isn't a keepass data file at all or it is corrupted." message? Or the "empty folder" issue described by [@gtedavid](https://github.com/gtedavid) ?

No. I must have misread the thread or picked the wrong one, as this is not the error I have. Sorry for this. 
I just described my issue here: https://github.com/PhilippC/keepass2android/issues/2919
